libdnf.spec

```diff
@@ -175,14 +184,14 @@ mkdir build-py3
 %build
 %if %{with python2}
 pushd build-py2
-  %cmake -DPYTHON_DESIRED:FILEPATH=%{__python2} -DWITH_MAN=OFF ../ %{!?with_valgrind:-DDISABLE_VALGRIND=1} %{_cmake_opts}
+  %cmake -DPYTHON_DESIRED:FILEPATH=%{__python2} -DWITH_MAN=OFF ../ %{?with_zchunk:-DWITH_ZCHUNK=ON} %{!?with_valgrind:-DDISABLE_VALGRIND=1} %{_cmake_opts}
```

Comment:
This should be `%{!?with_zchunk:-DWITH_ZCHUNK=OFF}` since the default is `ON` already...